tutorials/2d/2d_movement.rst

```diff
@@ -254,12 +254,9 @@ on the screen will cause the player to move to the target location.
 
         public override void _Input(InputEvent @event)
         {
-            if (@event is InputEventMouseButton eventMouseButton)
+            if (Input.IsActionPressed("click"))
```

Comment:
The `event` parameter should be used instead of the `Input` singleton in the `_input` callback. Also, it's missing the comment you added to the GDScript example.

```suggestion
            // Use IsActionPressed to only accept single taps as input instead of mouse drags.
            if (@event.IsActionPressed("click"))
```